yetanotherco/aligned_layer
# perf: reorder batchstate struct

> [!Warning]
> This PR will make previous batches unreadeable because it restructures the contract variables.

# To test:
Everything should work normally, so start:
- anvil, aggregator, operator, batcher, explorer
- send proofs
- they should
